### DISCUSSION TITLE
No module named 'nemo'

### DISCUSSION THREAD

**Discussion by jayachaturvedi** [NVIDIA-NeMo/NeMo#2950] [Q&A]
Hello, I wondered if anyone else gets this ModuleError: No module named 'nemo'? I have tried installing it using various methods mentioned on their docs page, but keep getting the same regardless of whether I try it locally or on colab. I have opened an issue, but wondered if I might get a response here if anyone else faced something similar?

Thanks!

J

**Comment by jaffe-fly** [comment]
maybe you dont install nemo correct,please check you nemo version

**Reply by jayachaturvedi** [reply]
What is the latest version that is available to install? I am following the installation guidelines in the documentation and it isn't working.
